requirements.txt

```diff
@@ -2,7 +2,7 @@ pytest>=3.0.4
 pytest-timeout
 pytest-random
 pysha3
-pyethapp>=1.5.0
+-e git+https://github.com/konradkonrad/pyethapp@71f940c6d287b98a35ef524f4c5e3c13d530bfc5#egg=pyethapp
```

Comment:
Maybe add a comment here linking to the PR: https://github.com/ethereum/pyethapp/pull/184 and explaining why this is just a temporary measure to be fixed as soon a new pyethap release is made?